components/logging/org.wso2.carbon.logging.service/src/main/java/org/wso2/carbon/logging/service/LoggingConstants.java

```diff
@@ -93,7 +93,7 @@ public final class LoggingConstants {
     public static final String CONNECTION_TIMEOUT = "connectionTimeout";
     public static final String REMOTE_SERVER_LOGGER_RESOURCE_PATH = "/identity/config/remoteServer";
 
-    public static final String AUDIT = "AUDIT";
-    public static final String CARBON = "CARBON";
+    public static final String AUDIT = "audit";
```

Comment:
Why we need this change to reflect URI endpoints

Reply:
In commons it has this condition check in the flows
https://github.com/wso2/carbon-commons/blob/bd8e0faa073cea3a2d0751136ce8e91680465c7c/components/logging/org.wso2.carbon.logging.service/src/main/java/org/wso2/carbon/logging/service/RemoteLoggingConfig.java#L148